python/lancedb/__init__.py

```diff
@@ -11,16 +11,24 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-from .db import URI, LanceDBConnection
+from typing import Optional
 
+from .db import URI, DBConnection, LanceDBConnection
+from .remote.db import RemoteDBConnection
 
-def connect(uri: URI) -> LanceDBConnection:
-    """Connect to a LanceDB instance at the given URI
+
+def connect(
+    uri: URI, api_key: Optional[str] = None, region: str = "us-west-2"
```

Comment:
Nit: let's add a * after uri requiring args to be named

Reply:
Fixed.